e2e/e2e_suite_test.go

```diff
@@ -46,7 +49,13 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		fmt.Printf("\nExposing Docker Registry\n")
 		<-testshell.Execute(`oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge`).Done()
 
+		<-testshell.Execute(NewProjectCmd(ImageRepo)).Done()
+		UpdateSecurityConstraintsFor(ImageRepo)
+
 		BuildOperator()
+		BuildTestService()
```

Comment:
Could we make these lazy instead? Would make it more convenient when running a single test to no have to build 'everything'.

Reply:
I can see how that could be done nicely - so far only ugly solutions are coming to mind.

Just wondering how frequently that would be a pain. Running a single test from the suite anyway will require to put `XDescribe` on all the other files but the one you do `FIt`, right? So in such a case you can just comment that one out too?

Reply:
You only need to do Fxx on the one you want, no need to Xxx anything. You can Focus from cli as well, but normally easier to just do it in code. 

I did the 'comment out' strategy on the auto complete stuff in global scope, so yea, doable, but ..

Reply:
As far as ugly, we only need a few global bools'.